pennylane/qchem/convert.py

```diff
@@ -564,8 +565,12 @@ def _ucisd_state(cisd_solver, tol=1e-15):
 def import_state(solver, tol=1e-15):
     r"""Convert an external wavefunction to a state vector.
 
+    Currently the only accepted source of wavefunctions is the PySCF library. The currently
+    supported methods are the restricted and unrestricted CISD/CCSD methods, with the function
+    accepting the corresponding Solver object from PySCF and inferring the type internally.
```

Comment:
Don't need to detail the inner workings of the function

```suggestion
    Currently the only accepted source of wavefunctions is the PySCF library. The currently
    supported methods are the restricted and unrestricted CISD/CCSD methods.
```